lib/v2/substack/index.js

```diff
@@ -0,0 +1,85 @@
+const path = require('path');
+const got = require('@/utils/got');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+
+const SUBSTACK_CATEGORIES = require('./substack-categories');
+const SUBSTACK_API_BASE_URL = 'https://substack.com/api/v1';
+
+/**
+ * This route allows subscribing Substack "Explore" categories, available via https://substack.com/browse.
+ *
+ * Substack currently does not offer a public developer API, so the internal API utilized on the Substack website is used here.
+ * Since it is not an "official" API, keep in mind that the API calls might break at any time.
+ */
+module.exports = async (ctx) => {
+    const { category, allowPaid = 'false', limit = '36' } = ctx.params;
```

Comment:
You can use the common query parameter `limit` from `ctx.query.limit`
docs: https://docs.rsshub.app/en/parameter.html#limit-entries